specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json

```diff
@@ -4406,6 +4410,14 @@
           "items": {
             "$ref": "#/definitions/ScriptAction"
           }
+        },
+        "virtualNetworkId": {
```

Comment:
When adding properties within an API version, there is risk if the properties are updated by a current version client, but then the resource is modified by a down-level client. This is not strictly a breaking change, but you certainly have more control against accidental overwrite if exposed in a new API version. What is the behavior on the service if an update excludes these properties after they have been previously defined?

Reply:
We are aware of these scenarios, and our customers also are used to these kinds of updates. We keep adding new features all the time, and they result in new properties, so adding new API version time is not feasible. The service overwrites the entities. This affects our SDK and Rest API users, but they are already aware of this, and keep getting new SDK versions when they want to use new properties. In fact, these changes are to catch up with our UX support.

Reply:
Ok, thanks